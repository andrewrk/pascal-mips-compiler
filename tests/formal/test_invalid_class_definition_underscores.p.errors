ERROR: line 8: illegal character '_' (0x5f)
