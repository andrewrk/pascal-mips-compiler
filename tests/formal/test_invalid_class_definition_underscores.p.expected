'_' (0137): illegal character at line 8
